tools/downloader/src/open_model_zoo/model_tools/converter.py

```diff
@@ -104,11 +109,30 @@ def main():
 
     args = parser.parse_args()
 
+    try:
+        telemetry = tm.Telemetry('Model Converter', tid='UA-194864834-1')
+        telemetry.start_session('md')
+
+        if args.all:
+            telemetry.send_event('md', 'converter_selection_mode', 'all')
+        if args.list:
+            telemetry.send_event('md', 'converter_selection_mode', 'list')
+        if args.name:
+            telemetry.send_event('md', 'converter_selection_mode', 'name')
+    except NameError:
+        pass
```

Comment:
All these exception handlers are noisy and could mask `NameError`s that are unrelated to telemetry. I would suggest to instead add something like this to `_common`:

```python
try:
    from openvino_telemetry import Telemetry
except ImportError:
    class Telemetry:
        def __init__(self, tool_name, tid): pass
        ...
```

Then you just import `Telemetry` from `_common` and use it without any error handling.

Reply:
Done